ding2.install

```diff
@@ -1042,3 +1042,10 @@ function ding2_update_7089() {
 function ding2_update_7090() {
   ding2_set_eu_cookie_compliance_settings();
 }
+
```

Comment:
We should not enable it by default. Also, we do not change ding2.install file. If we do anything we use easyddb_frontend_layout.install.

Reply:
+